uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -545,13 +625,166 @@ define([
       /**
        * checks if all params are set to calculate balance
        *
-       * @param {Array} listOfDayTypes collection of available day types
+       * @param {Boolean} true if all present else false
        */
       function canCalculateChange() {
         return vm.leaveRequest.from_date && vm.leaveRequest.to_date &&
           vm.leaveRequest.from_date_type && vm.leaveRequest.to_date_type;
       }
 
+      /**
+       * checks if leaverequest is managed by given manager id
+       *
+       * @param {String} managerContactId
+       * @returns {Promise}
+       */
+      function isManagedBy(managerContactId) {
```

Comment:
an `isSomething` function is expected to return a boolean value, but here instead it ends up with setting the value of `isManager`. So either you change the name of the function, or you change the implementation